src/classes/CRLP_RollupCMT.cls

```diff
@@ -122,6 +134,36 @@ public class CRLP_RollupCMT {
             }
             return rulesMDT;
         }
+
+    /******************************************************************************************************************
+     * @description Determines if a Filter Group is computationally different from another Filter Group
+     * changes in name or label or description are not considered different
+     * filter rules are hashed because the sort order of the list is not guaranteed
+     * @param otherFilterGroup FilterGroup to compare against
+     * @return boolean to indicate of the filter group is different 
+     *****************************************************************************************************************/
+        public Boolean isDifferent(FilterGroup otherFilterGroup) {
```

Comment:
Yeah I thought about using that - my hesitation is that - it is not really an equal comparison.  I'm not checking if the labels/descriptions are the same - so the two objects could have different values yet not be different - computationally.  But I'm open to making that change since It seems like that is the intent.

Reply:
Oh that's a good point, let's leave this as is then. Implementing equals and hashCode may be more trouble than it's worth